data/json/items/ammo.json

```diff
@@ -749,6 +749,51 @@
         "ammo_type" : "rebreather_filter",
         "count" : 30
     },
+    {
+        "type": "AMMO",
+        "id": "gasfilter_s",
+        "category": "spare_parts",
+        "price": 2500,
+        "name": "filter mask cartridge",
+        "symbol": "=",
+        "color": "light_gray",
+        "description": "Small replacement filter cartridge for air filtration masks.",
+        "material": "wood",
```

Comment:
Again, remove the `"material": "wood"` field for all filter cartridges.

Reply:
Forgot to remove it from the last version.